### ISSUE TITLE
[Feature request]: Grid Layout Template

### ISSUE THREAD

**Issue by o-data** [vzhd1701/gridplayer#121]
### Suggestion

Currently GridPlayer just stack vertically and horizontally, which is great!

The feature request is to have a template of various layout and setup, either customize or pre-templated. If you use those Camera/DVR App or viewer that we layout various video in different way.

Here is a screenshot as an example.
https://i.pinimg.com/736x/d0/69/41/d0694161ddd8c9ea709a5d8449e08b08--remote-viewing-ip-camera.jpg

Basically for various grid size. Case use: 1 large grid for focus, with the rest small for additional view.

The first photo is "Templated" Grid size so it all Static. 

Another example is this software which we can set big and small grid as we: https://www.suu-design.com/Files/VTM/Anim_eng.png
